sparsity/io.py

```diff
@@ -11,6 +12,30 @@
 except (ImportError, OSError):
     TrailDB = False
 
+
+class LocalFileSystem():
+
+    open = open
+
+FILE_SYSTEMS = {
+    '': LocalFileSystem,
+    'file': LocalFileSystem
+}
+
+try:
+    import s3fs
```

Comment:
There's already `s3fs` import above. Wrapping this one in `try` doesn't seem to make sense.

Reply:
true I'll remove the one above

Reply:
Forgot about this one? :P